### PR TITLE
[Merged by Bors] - chore: rename `condCount` to `uniformOn`

### PR DESCRIPTION
**Pull request by YaelDillies**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Probability/CondCount.lean [hunk 1]
```diff
@@ -21,15 +21,15 @@ for that purpose.
```
Comment (sgouezel) [inline Mathlib/Probability/CondCount.lean:21]: Can you rename the file to `UniformOn`?

File: Archive/Wiedijk100Theorems/BirthdayProblem.lean [hunk 2]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Eric Rodriguez
 -/
 import Mathlib.Data.Fintype.CardEmbedding
-import Mathlib.Probability.CondCount
+import Mathlib.Probability.uniformOn
```
Comment (sgouezel) [inline Archive/Wiedijk100Theorems/BirthdayProblem.lean:7]:
```suggestion
import Mathlib.Probability.UniformOn
```

File: Mathlib/Probability/CondCount.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Probability/CondCount.lean:21]: I was afraid it would mess the diff, but looks like it went fine

Comment (YaelDillies) [reply Mathlib/Probability/CondCount.lean]: Nevermind. It did mess up the diff. Are you sure you want me to do this?

File: Mathlib/Probability/UniformOn.lean [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+open ProbabilityTheory
+
+open MeasureTheory MeasurableSpace
+
+namespace ProbabilityTheory
+
+variable {Ω : Type*} [MeasurableSpace Ω]
+
+/-- Given a set `s`, `uniformOn s` is the counting measure conditioned on `s`. In particular,
```
Comment (b-mehta) [inline Mathlib/Probability/UniformOn.lean:47]: With the new name, adding something to this docstring to justify the subterm "uniform" would be nice.

**Comment by RemyDegenne** [comment]
LGTM, thanks!
bors r+

**Pull request closed without merging**